man/systemd.network.xml

```diff
@@ -1631,7 +1631,7 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
           <listitem>
             <para>When true, the routes to the DNS servers received from the DHCP server will be
             configured. When <varname>UseDNS=</varname> is disabled, this setting is ignored.
-            Defaults to false.</para>
+            Defaults to true.</para>
```

Comment:
doesn't matter, but i think we should attempt to standardize on spelling the boolean values out as yes/no in our docs, and not true/false or 1/0, even though we accept all three ways. it's the closest to spoken language i think. And I think most of our docs even already use yes/no, but far from all

but treally doesn't matter.